database_wrapper.py

```diff
@@ -863,18 +654,16 @@ def __del__(self):
 
 if __name__ == "__main__":
     db = NimbusMySQLAlchemy()
-
     db._create_all_tables()
 
     data = {
         "building_number": 1,
         "name": "Administration",
         "longitude": -120.658561,
-        "latitude": 35.300960,
+        "latitude": 35.300960
     }
 
-    db.save_location(data)
-
+    db.save_entity(Locations, data)
```

Comment:
lets rememeber to use these tests in the `/tests/` folder with pytest && some mocking library